recipes/taxoniq/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "taxoniq" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 19fb5d7f85158dc9e80c636635716ce14bb13671abefc9b785257789353f6982
+
+build:
+  number: 0
+  entry_points:
+    - taxoniq=taxoniq.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - urllib3 >=1.25.8
+    - zstandard >=0.15.1
+  run:
+    - python
+    - urllib3 >=1.25.8
+    - zstandard >=0.15.1
+
+test:
+  imports:
+    - taxoniq
+    - taxoniq.vendored
+  commands:
+    - taxoniq --help
+  requires:
+    - coverage
+    - flake8
+    - wheel
```

Comment:
```suggestion
    - taxoniq --help
    - pip check
  requires:
    - pip
```

Reply:
coverage and flake8 are development libraries. we don't need to lint code at this stage.

The importance of pip check is explained in https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L58